core/modules/flowgen.cc

```diff
@@ -200,6 +200,13 @@ pb_error_t FlowGen::ProcessArguments(const bess::pb::FlowGenArg &arg) {
     quick_rampup_ = 1;
   }
 
+  if (arg.burst() <= bess::PacketBatch::kMaxBurst) {
```

Comment:
Another delicate point: `Init()` should set burst to 32 (default) if 0 is given, since it may have been unspecified in protobuf. `set_burst()` should accept 0 (generating no more packets) since it is explicitly given.